usr/share/rear/output/USB/Linux-i386/100_create_efiboot.sh

```diff
@@ -95,8 +95,12 @@ EOF
             DebugPrint "Configuring GRUB2 for EFI boot"
             # We need to explicitly set GRUB 2 'root' variable to $efi_label (hardcoded "REAR-EFI")
             # because default $root would point to memdisk, where kernel and initrd are NOT present.
-            # GRUB2_SEARCH_ROOT_COMMAND is used in the create_grub2_cfg() function:
-            [[ -z "$GRUB2_SEARCH_ROOT_COMMAND" ]] && GRUB2_SEARCH_ROOT_COMMAND="search --no-floppy --set=root --label $efi_label"
+            # GRUB2_SEARCH_ROOT_COMMAND is used in the create_grub2_cfg() function.
+            # Because GRUB2_SEARCH_ROOT_COMMAND is empty in default.conf
+            # we can provide final power to the user and always respect
+            # GRUB2_SEARCH_ROOT_COMMAND when the user has set it
+            # cf. https://github.com/rear/rear/issues/3024#issuecomment-1630963778
+            test "$GRUB2_SEARCH_ROOT_COMMAND" || GRUB2_SEARCH_ROOT_COMMAND="search --no-floppy --set=root --label $efi_label"
```

Comment:
Shouldn't that be `contains_visible_char` (same for all the others)?

Reply:
I won't do a general cleanup via this pull request,
cf. https://github.com/rear/rear/pull/3025#issuecomment-1635340530